manifests/repo.pp

```diff
@@ -34,8 +34,13 @@
     }
     case $facts['os']['family'] {
       'RedHat': {
-        $gpgkey_zabbix = 'https://repo.zabbix.com/RPM-GPG-KEY-ZABBIX-A14FE591'
-        $gpgkey_nonsupported = 'https://repo.zabbix.com/RPM-GPG-KEY-ZABBIX-79EA5ED4'
+        if $facts['os']['release']['major'] > '8' {
```

Comment:
Use `versioncmp` here

Reply:
+1

```puppet
versioncmp(fact('os.release.full'), '8.0') >= 0
```

Reply:
Can we move this along? I've tried checking out the PR and fixing this myself but I don't have github set up properly as I don't use it often.

Reply:
Can we move this along? :-) I tried checking out the PR and updating it but I don't seem to have the permissions to do this.